CMakeLists.txt

```diff
@@ -53,16 +54,16 @@ set(ENABLE_STATIC_LINK ON  CACHE BOOL "Enable static linking?")
 set(ENABLE_STRIP       ON  CACHE BOOL "Enable stripping all symbols from release binary?")
 
 # Option overrides
-if(JERRY_CMDLINE OR JERRY_CMDLINE_MINIMAL)
+if(JERRY_CMDLINE OR JERRY_CMDLINE_MINIMAL OR UNITTESTS OR DOCTESTS)
```

Comment:
Both unit-core and unit-ext depends on the jerry-port-default-minimal library. However, somehow this dependency did not appear here until now. (Actually, run-tests.py hid it, because it did not turn off cmdline when building the unit tests, and cmdline forced the building of the default port. So, we have never tried building unittests alone, without the cmdline. If we had, we would have run into a link issue before, too.) So, this is actually an independent addition, but as this line is already touched by this PR, I thought it was OK to fix this little problem here.